app/src/main/java/chat/rocket/android/helper/RocketChatUserAvatar.kt

```diff
@@ -1,17 +1,91 @@
 package chat.rocket.android.helper
 
+import android.content.Context
+import android.graphics.Typeface
+import android.graphics.drawable.Drawable
+import com.amulyakhare.textdrawable.TextDrawable
 import java.net.URLEncoder
 
-class RocketChatUserAvatar(val hostname: String, val username: String) {
-
-    val imageUri: String
-        /** REMARK
-         * This is often a SVG image (see Rocket.Chat:server/startup/avatar.js).
-         */
-        get() {
-            return "https://" +
-                    hostname.replace("http://", "").replace("https://", "") +
-                    "/avatar/" +
-                    URLEncoder.encode(username, "UTF-8")
+object RocketChatUserAvatar {
+
+    /**
+     * Returns the user avatar URI.
+     *
+     * REMARK: This is often a SVG image (Rocket.Chat:server/startup/avatar.js).
+     *
+     * @param hostname The server's hostname.
+     * @param username The username.
+     * @return The user avatar URI.
+     * @see getImageFormat
+     */
+    fun getUri(hostname : String, username: String): String {
+        return "https://" +
+                hostname.replace("http://", "").replace("https://", "") +
+                "/avatar/" +
+                URLEncoder.encode(username, "UTF-8")
+    }
+
+    /**
+     * Returns the user avatar image format.
+     *
+     * @param uri The user avatar image URI to get the image format.
+     * @return The user avatar image format. Possible values are: "image/jpeg", "image/png" and "image/svg+xml".
+     * @see getUri
+     */
+    fun getImageFormat(uri: String): String {
+        return OkHttpHelper.getContentType(uri)
```

Comment:
Yes, I pay attention w/ it, so to get the image format the requested method is `HEAD` and not `GET`. Take a [look](https://github.com/filipedelimabrito/Rocket.Chat.Android/blob/281ac8a39afa664b04cc3bf0e0674818f049682c/app/src/main/java/chat/rocket/android/helper/OkHttpHelper.java).

https://www.w3.org/Protocols/rfc2616/rfc2616-sec9.html:
> The HEAD method is identical to GET except that the server MUST NOT return a message-body in the response. The metainformation contained in the HTTP headers in response to a HEAD request SHOULD be identical to the information sent in response to a GET request. This method can be used for obtaining metainformation about the entity implied by the request without transferring the entity-body itself. This method is often used for testing hypertext links for validity, accessibility, and recent modification.